PiwikTracker/Device.swift

```diff
@@ -14,20 +16,30 @@ public struct Device {
     }
     
     /// The platform name of the device i.e. "iPhone1,1" or "iPad3,6"
-    public let platform: String
+    @objc public let platform: String
     
     /// A human readable version of the platform name i.e. "iPhone 6 Plus" or "iPad Air 2 (WiFi)"
     /// Will be nil if no human readable string was found.
-    public let humanReadablePlatformName: String?
+    @objc public let humanReadablePlatformName: String?
     
     /// The version number of the OS as String i.e. "1.2" or "9.4"
-    public let osVersion: String
+    @objc public let osVersion: String
     
     // The screen size
-    public let screenSize: CGSize
+    @objc public let screenSize: CGSize
     
     // The native screen size
-    public let nativeScreenSize: CGSize?
+    @objc public let nativeScreenSize: CGSize
```

Comment:
You changed the nativeScreenSize to non-optional, but for macOS it is not defined. Correct me if I am wrong on that point.

Reply:
My issue was that optional non class parameter like `CGSize?` cannot be represented in objc. I got a build error when prefixing it with `@objc`.
As a Swift rookie, I decided to remove the optionality. Could you advice what is the best thing to do in that situation?

Reply:
Ah, I see your issue there. Yeah. Only pointers to objects can be represented as optionals in Objective-C (null pointer then). 

Since making this a pointer would be very weird I think there would be two solutions. Either define "no native screen size is defined" as `CGSize.zero` or as "is equivalent to screen size". I think the first would be better, since a screen size with no width or height could easily be understood as "not defined". Either way, please add a short comment in the documentation.

Reply:
Lovely. Thanks!